source/client-side-encryption/client-side-encryption.rst

```diff
@@ -638,6 +638,9 @@ server selection on the MongoClient to mongocryptd fails. If the
 MongoClient fails to connect after spawning, the server selection error
 is propagated to the user.
 
+Single-threaded drivers MUST connect with `serverSelectionTryOnce=false <https://github.com/mongodb/specifications/blob/6bc8afe3516d2274c3f646e8293af15cf8651e8c/source/server-selection/server-selection.rst#serverselectiontryonce>`_
```

Comment:
Most cross-spec links use relative URIs, so I think this could be `../server-selection/server-selection.rst#serverselectiontryonce`. Likewise for the other links in this PR.

Reply:
Done.